extensions/homie extension convention.md

```diff
@@ -0,0 +1,61 @@
+# The Homie Extension Convention
```

Comment:
Yes, it should somehow be part of the main spec. But I think we should try avoid getting the main spec as seen on the website to long. As form my personal experience, I decided to take a closer look at Homie because the convention page on the website is relativly short compared to other specifications. What exactly an extension is might not be that interesting for new users who just want to get a quick idea of the convention. Althrough knowing that there are extensions is important ofcourse. So I suggest we should try to make a separate tab on the website.

Reply:
I'm with you. I do not suggest to put the entire text into the main spec. Only the license part, so one sentence. And the rest goes to the webpage. But nobody will ever go this repository into the documents subdirectory to find "homie extension convention.md" I guess.